spec/docbase_spec.rb

```diff
@@ -0,0 +1,143 @@
+          <p>
+          <span><img src=\"https://secure.gravatar.com/avatar/9922bb39304a94c92e862211c23d60fc.png\" width=\"16\" height=\"16\" alt=\"\" /></span>
+          <a href='https://idobata.docbase.io/users/1'>eiwakun</a>
+          updated <a href='https://idobata.docbase.io/posts/1'>hookの作り方</a>
+          at idobata
+          </p>
+        HTML
+    end
+
+    describe 'post_group_update event' do
+      let(:event_type) { 'post_group_update' }
+
+      it { expect(subject[:source]).to be_dom_equal <<~HTML }
+          <p>
+          <span><img src=\"https://secure.gravatar.com/avatar/9922bb39304a94c92e862211c23d60fc.png\" width=\"16\" height=\"16\" alt=\"\" /></span>
+          <a href='https://idobata.docbase.io/users/1'>eiwakun</a>
+          post group updated <a href='https://idobata.docbase.io/posts/1'>hookの作り方</a>
+          at idobata
+          </p>
+        HTML
+    end
+
+    describe 'sharing_create event' do
+      let(:event_type) { 'sharing_create' }
+
+      it { expect(subject[:source]).to be_dom_equal <<~HTML }
+          <p>
+          <span><img src=\"https://secure.gravatar.com/avatar/9922bb39304a94c92e862211c23d60fc.png\" width=\"16\" height=\"16\" alt=\"\" /></span>
+          <a href='https://idobata.docbase.io/users/1'>eiwakun</a>
+          shared <a href='https://idobata.docbase.io/posts/1'>hookの作り方</a>
+          at idobata
+          </p>
+        HTML
+    end
+
+    describe 'sharing_destroy event' do
+      let(:event_type) { 'sharing_destroy' }
+
+      it { expect(subject[:source]).to be_dom_equal <<~HTML }
+          <p>
+          <span><img src=\"https://secure.gravatar.com/avatar/9922bb39304a94c92e862211c23d60fc.png\" width=\"16\" height=\"16\" alt=\"\" /></span>
+          <a href='https://idobata.docbase.io/users/1'>eiwakun</a>
+          sharing destroyed <a href='https://idobata.docbase.io/posts/1'>hookの作り方</a>
+          at idobata
+          </p>
+        HTML
+    end
+
+    describe 'comment_create event' do
+      let(:event_type) { 'comment_create' }
+
+      it { expect(subject[:source]).to be_dom_equal <<~HTML }
+          <p>
+          <span><img src=\"https://secure.gravatar.com/avatar/9922bb39304a94c92e862211c23d60fc.png\" width=\"16\" height=\"16\" alt=\"\" /></span>
+          <a href='https://idobata.docbase.io/users/1'>eiwakun</a>
+          commented on <a href='https://idobata.docbase.io/posts/1'>hookの作り方</a>
+          at idobata
+          </p>
+          <p>コメントテスト</p>
+        HTML
+    end
+
+    describe 'star_create event' do
+      let(:event_type) { 'star_create' }
+
+      it { expect(subject[:source]).to be_dom_equal <<~HTML }
+          <p>
+          <span><img src=\"https://secure.gravatar.com/avatar/9922bb39304a94c92e862211c23d60fc.png\" width=\"16\" height=\"16\" alt=\"\" /></span>
+          <a href='https://idobata.docbase.io/users/1'>eiwakun</a>
+          starrd <a href='https://idobata.docbase.io/posts/1'>hookの作り方</a>
```

Comment:
seems misspelling for "starred" :star2:

Reply:
😇 
Thanks for review!
I fixed it and force pushed. ⭐️